store/README.md

```diff
@@ -0,0 +1,167 @@
+# Store
+
+**Store** maintains the state of the chain. It serves as the "source of truth" for the chain - i.e., if it is not in 
+the store, the node does not consider it to be a part of the chain. 
+**Store** is one of components of the [Miden node](..).
+
+## Architecture
+
+The Miden node is still under heavy development and current architecture is subject of change. This topic will be 
+filled later.
+
+## Usage
+
+### Installing the Store
+
+Store can be installed and run as a part of [Miden node](../README.md#installing-the-node).
```

Comment:
Change to: The Store can be installed and run as part of [Miden node](https://github.com/0xPolygonMiden/miden-node/pull/README.md#installing-the-node).
But if you intend on running the Store as a separate process, you will need to install and run it as follows: